### DISCUSSION TITLE
Deprecation of Vue and Solid integrations

### DISCUSSION THREAD

**Discussion by mattgperry** [motiondivision/motionone#241] [Announcements]
Hi everyone!

A bit of bad news to start the year 😅

I'm deprecating the Vue and Solid integrations of Motion One. The reason being, they have low usage next to the core package. Between work and parenthood I have quite little time for side-projects so I haven't been able to support these to the standard I'd prefer.

With Motion One it's quite easy to justify the little time I do spend on it as improvements to it often end up in Framer Motion (and vice versa!) However with the view library integrations I can't say the same.

I've been meaning to do this for quite a while but currently some dependency issue is blocking [`autoplay`](https://github.com/motiondivision/motionone/pull/239) so I decided to pull the trigger today.

A [PR to remove these packages](https://github.com/motiondivision/motionone/pull/240) will be merged next week.

In the meantime I would encourage anyone who has an interest in keeping these packages to fork the code and release their own library. I'll gladly link to your docs from `motion.dev/vue` and `motion.dev/solid` when you have them ready!

**Comment by productdevbook** [comment]
[oku-ui/motion](https://github.com/oku-ui/motion)

I'll provide Vue and Nuxt support from here, don't worry. ❤️

**Reply by productdevbook** [reply]
@mattgperry [oku-ui.com/motion](https://t.co/YWPpQ31e9b)

**Reply by mattgperry** [reply]
Great, thanks for taking this on!

**Comment by thetarnav** [comment]
Happy new year :smile: 
I think that's a good decision, I'm sure it will benefit the development of both motionone and framework integrations.
In the meantime we've moved the existing Solid integration to [solidjs-community/solid-motionone](https://github.com/solidjs-community/solid-motionone).

**Reply by mattgperry** [reply]
Amazing news @thetarnav !

**Reply by thetarnav** [reply]
@mattgperry could you deprecate the `@motionone/solid` package on npm, with a link to this discussion?

**Reply by mattgperry** [reply]
Sure well I've deprecated it with a link to your repo instead
